src/CitizenProject/CitizenProjectCommand.php

```diff
@@ -185,4 +192,23 @@ public function setCitizenProject(CitizenProject $citizenProject): void
     {
         $this->citizenProject = $citizenProject;
     }
+
+    public function getSkills(): Collection
+    {
+        return $this->skills;
+    }
+
+    public function setSkills(iterable $skills): void
```

Comment:
IMO, `set` means "ok let's start over with this data". Here it's not the case. It's an `addSkills` method.
So either rename it or modify it to match the semantic of the method.

Reply:
If I remove all elements in $this->skills before call for each, it's good ?

Reply:
Yea I think so. You can also create a brand new Collection then add the one provided.